teuthology/orchestra/remote.py

```diff
@@ -599,6 +598,22 @@ def sudo_write_file(self, path, data, **kwargs):
         """
         self.write_file(path, data, sudo=True, **kwargs)
 
+    def write_temp_file(self, **kwargs):
```

Comment:
```suggestion
    def write_temp_file(self, suffix=None, parentdir=None, **kwargs):
```

Reply:
Thanks for the review. I don't understand why accepting them in kwargs is not good enough. Also, an disadvantage of this style of parameters is that callers of `write_temp_file()` can end up passing multiple values for a parameter which is an unnecessary bug. Just `**kwargs` is better IMHO.

Reply:
> can end up passing multiple values for a parameter which is an unnecessary bug.

example?

Reply:
```
>>> def somefunc(a, b, c, d, e):
...     pass
... 
>>> somefunc(1, 2, 3, 4,**kw)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: somefunc() got multiple values for argument 'c'
>>> 
```
Not mixing leads to a code that is more sturdy, lesser chance for this sort of bug.